src/components/Login.js

```diff
@@ -115,11 +119,12 @@ class Login extends Component {
       <>
         <Grid>
           <Grid.Row>
+            <div className="preloader"> </div>{' '}
```

Comment:
Please Remove empty string brackets.

```suggestion
            <div className="preloader"> </div>
```